sql/hive-thriftserver/pom.xml

```diff
@@ -69,17 +69,21 @@
       <artifactId>hive-beeline</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.sun.jersey</groupId>
-      <artifactId>jersey-core</artifactId>
+      <groupId>org.glassfish.jersey.core</groupId>
```

Comment:
I know you're just doing a find / replace here, but are these dependencies even needed in this module? I don't see any explicit references to jersey in the code.